OpenSim/Tools/AnalyzeTool.cpp

```diff
@@ -434,8 +442,14 @@ setStatesFromMotion(const SimTK::State& s, const Storage &aMotion, bool aInDegre
     _model->getSimbodyEngine().convertDegreesToRadians(*qStore);
     _model->getSimbodyEngine().convertDegreesToRadians(*uStore);
 
-    _statesStore = createStatesStorageFromCoordinatesAndSpeeds(*_model, *qStore, *uStore);
+    double ti = qStore->getFirstTime();
+    double tf = qStore->getLastTime();
+    uStore->addToRdStorage(*qStore, ti, tf);
+
+    delete _statesStore;
+    _statesStore = new Storage(512, "states");
 
+    _model->formStateStorage(*qStore, *_statesStore);
     delete qStore;
```

Comment:
It should be changed for consistency as well. Why it likely doesn't matter is that the `AnalyzeTool` is used to run the old `InverseDynamics` `Analysis` to compute residuals. The model in RRA typically replaces its `ForceSet` and therefore has no muscles, in which case there are no states other than coordinates and their speeds. 

Removing the redundancy of `AnalyzeTool::createStatesStorageFromCoordinatesAndSpeeds()` is a small improvement. I can make it part of this PR and or do that separately. Separate may be better since this PR's aims to fix the bug in #1675.